lib/std/sha1.nim

```diff
@@ -170,23 +195,46 @@ proc finalize(ctx: var Sha1State): Sha1Digest =
 # Public API
 
 proc secureHash*(str: string): SecureHash =
+  ## Generates a SecureHash from a string and returns it.
```

Comment:
I would have either "Generates and returns a `SecureHash` from a string `str`", or remove "returns" completely: "Generates a `SecureHash` from a string `str`"

----

I'll repeat a comment from `uri`:

General remark: I would love to see "See also:" section which will give links to similar procs. Some examples: [strutils.indent](https://nim-lang.github.io/Nim/strutils.html#indent%2Cstring%2CNatural%2Cstring), [tables.getOrDefault](https://nim-lang.github.io/Nim/tables.html#getOrDefault%2CTable[A%2CB]%2CA), etc.

Here, specifically, a link to `secureHashFile` and `parseSecureHash` would be useful. (And vice-versa)

Reply:
Thank you - and I'm also convinced of that ;)